source/speechDictHandler/__init__.py

```diff
@@ -103,10 +109,12 @@ def sub(self, text):
 		return text
 
 def processText(text):

```

Comment:
Could you add unit tests for this code? It would be good to formally show the cases supported and not supported.

Reply:
In particular, it would be good to get automated testing for the number processing, essentially covering the changes in this PR.